code/game/objects/items/storage/boxes.dm

```diff
@@ -152,8 +156,36 @@
 	for(var/i in 1 to 7)
 		new /obj/item/disk/nanite_program(src)
 
-// Ordinary survival box
+//Parent box to accomodate station trait and apply unique restrictions
+/obj/item/storage/box/survival
+	name = "survival box"
+	illustration = "survival"
+	desc = "A compact box that is designed to hold specific emergency supplies"
+	w_class = WEIGHT_CLASS_SMALL //So the roundstart box takes up less space.
+
+/obj/item/storage/box/survival/Initialize(mapload)
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_items = 5
+	STR.max_combined_w_class = 21
+	var/static/list/can_hold = typecacheof(list(
```

Comment:
I kind of dislike this change of the box not being able to hold other various small items like pills and stuff since it could always be handy for carrying around some extra tiny items. Especially since backpacks have not only a limit on max weight but max item count too, giving boxes some use even though they are harder to access in the heat of battle adding a delay to quickly getting stuff you have stored on you. I don't see why the roundstart box should be hugely different from ones that you can find being large and can hold any items as long as they are pretty small items

Reply:
> I kind of dislike this change of the box not being able to hold other various small items

It can be turned into a normal box through an admittedly somewhat obscure method: Break the box down into cardboard and then use the cardboard to make a normal box with large capacity and size but no item restrictions.

I figured the majority of players would prefer the ability to fit more loose objects in their backpacks, and thought it was more intuitive overall this way. Players with an excessive amount of pills likely have access to pill bottles, for instance, and cardboard isn't prohibitively difficult to get either.

Reply:
> Especially since backpacks have not only a limit on max weight but max item count too

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/9547572/0f6b2569-ab81-4cf7-9fc9-e83f13e22473)

The item limit is the same as their capacity limit for a standard backpack. Only a duffelbag will ever hit this limit and it is highly unlikely someone will use a duffelbag to hold more than 28 tiny items.